pilot/prompts/architecture/technologies.prompt

```diff
@@ -33,7 +33,7 @@ Here are user tasks that specify what users need to do to interact with "{{ name
 ```
 
 {% endif %}
-Now, based on the app's description, user stories and user tasks, think step by step and list the names of the technologies that will be used by your development team to create the app "{{ name }}". Do not write any explanations behind your choices but only a list of technologies that will be used.
```

Comment:
Not sure why is this removed but I guess this works now because of function calling.

Reply:
This prompt was written when we didn't have function calling and it was returning string. Besides that, second big thing is that we are saying same thing multiple times, for example, bellow there is another line:

`Do not give any subjective thoughts or options, give exact technologies, for example:`

That makes it harder to understand, debug and change